source/user-manual/reference/tools/manage_agents.rst

```diff
@@ -53,9 +53,15 @@ Options
 |               +-----------------------------+-----------------------------------------+
```

Comment:
The **Usage** section must be updated: 
- The **[-F sec]** parameter no longer exists
- The **[-R sec]** and **[-D sec]** options should be added

Reply:
I missed that. Thanks!